sev/src/firmware/linux/ioctl/wrapper.rs

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use std::convert::TryInto;
+use std::io::Error;
+use std::marker::PhantomData;
+use std::os::raw::{c_int, c_uint, c_ulong, c_void};
+use std::os::unix::io::AsRawFd;
+use std::ptr::null;
+
+extern "C" {
+    pub fn ioctl(fd: c_int, request: c_ulong, ...) -> c_int;
+}
+
+#[repr(u32)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub enum Code {
+    Init = 0,
+    EsInit,
```

Comment:
~~EsInit~~ -> InitEx

Reply:
Since these are ioctls to /dev/kvm, it might be a good idea for the names to be similar. For example, this is KVM_SEV_ES_INIT in kvm.h. What do you think ?

Reply:
Ahh I'm torn now. On one hand I prefer `InitEx` because it aligns with the SEV API spec and I don't want to deviate from it just because the kernel does...

On the other hand, leaving it as is is technically more consistent with the interface we're using (the one exported by the kernel).

Let's leave it as `EsInit`, but please leave a comment saying `/* AKA InitEx in the SEV spec */`